### PR TITLE
Tkt 00038

### PR DESCRIPTION
**Pull request by Android63**

### CODE REVIEW

File: style/headers.scss [hunk 1]
```diff
@@ -0,0 +1,6 @@
+/**
+* Remove bottom margin on headers
+*/
+h1, h2 {
+  margin-bottom: 0;
```
[the remark this answers is not in the document]
Comment (Android63) [reply style/headers.scss]: I agree. Styling like this, if needed, should be in core Nightingale and so I have removed this.

File: style/images.scss [hunk 2]
```diff
@@ -0,0 +1,11 @@
+/**
+* Restrict images to width of container
+* with height adjusting to maintain aspect ratio
+*/
+img {
+  max-width: 100% !important;
```
[the remark this answers is not in the document]
Comment (Android63) [reply style/images.scss]: I have removed this. If image styling is needed, it should be in core Nightingale.

**Comment by Android63** [comment]
As shown in the screenshots below, this change is to allow admins to display posts in full, as excerpts, or as titles only:

screen shot 2018-01-17 at 10 31 55
screen shot 2018-01-17 at 10 32 21
screen shot 2018-01-17 at 10 32 41

Could you please review this @cehwitham?

File: .gitignore [hunk 3]
```diff
@@ -34,6 +34,7 @@ wp-config.php
 /wp-content/uploads/*
 /wp-content/mu-plugins/load-wp-performance-profiler.php
 /wp-content/mu-plugins/wp-performance-profiler/*
+test.html
```
Comment (cehwitham) [inline .gitignore]: Should this be here?

File: home.php [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+	<div id="primary" class="o-layout__item u-8/12@lg">
+		
+		<main id="main" class="site-main" role="main">
+
+		<?php
+		if ( have_posts() ) :
+
+			if ( ! is_front_page() ) : ?>
+				<header>
+					<h1 class="page-title screen-reader-text"><?php single_post_title(); ?></h1>
```
Comment (cehwitham) [inline home.php]: What title is this outputting. If it outputs when it is not the homepage, this template will be used as a blog listing template so not sure why we need a single_post_title here?

File: .gitignore (hunk 3 above)
Comment (Android63) [reply .gitignore]: Yes, I use this to test out various markups locally but don't want it included in the main repo.

**Pull request merged**